requirements.txt

```diff
@@ -1,12 +1,44 @@
-Flask==0.10.1
-Jinja2==2.7.3
-MarkupSafe==0.23
-Werkzeug==0.9.6
-gnureadline==6.3.3
-itsdangerous==0.24
-rauth==0.7.0
-requests==2.3.0
-wsgiref==0.1.2
-
-gunicorn==18.0
-Flask-SSLify==0.1.4
+aniso8601==7.0.0
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2021-33503   
**Check Name:** python-urllib3: ReDoS in the parsing of authority part of URL   
**Severity:** HIGH   
**Fixed Version:** 1.26.5   
**Description:** An issue was discovered in urllib3 before 1.26.5. When provided with a URL containing many @ characters in the authority component, the authority regular expression exhibits catastrophic backtracking, causing a denial of service if a URL were passed as a parameter or redirected to via an HTTP redirect.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2021-33503